app/models/manageiq/providers/vmware/infra_manager/vm/reconfigure.rb

```diff
@@ -1,7 +1,7 @@
 module ManageIQ::Providers::Vmware::InfraManager::Vm::Reconfigure
   # Show Reconfigure VM task
   def reconfigurable?
-    true
+    !archived?
```

Comment:
Prefer `active?` over `!archived?` because a vm could be `orphaned?` and `!archived?`